docs/source/examples/demo_tracing_model_torchscript_onnx.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fba4c02d",
+   "metadata": {},
+   "source": [
+    "# Demo Notebook to trace Sentence Transformers model\n",
+    "\n",
+    "This notebook provides a walkthrough guidance for users to trace models from Sentence Transformers in torchScript and onnx format. After tracing the model customer can upload the model to opensearch and generate embeddings.\n",
+    "\n",
+    "Step 0: Import packages and set up client\n",
+    "\n",
+    "Step 1: Save model in torchScript format\n",
```

Comment:
Will steps be confusing for user? They may think they need to go through all steps to generate embeddings. How about split it into two options? We can tune line 10 to let user know they can upload torch_sciprt or ONNX model, either option is ok, then explain two options in details.

Reply:
Sure, makes sense. I'll try to clarify more in the description.